hwdb/60-evdev.hwdb

```diff
@@ -267,6 +267,13 @@ evdev:name:AlpsPS/2 ALPS DualPoint TouchPad:dmi:*svnDellInc.:pnPrecisionM4700*
  EVDEV_ABS_35=0:1960:24
  EVDEV_ABS_36=113:1436:30
 
+# Dell XPS13 9360
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnDellInc.:pnXPS139360:cvr:*
+ EVDEV_ABS_00=1276:5666:42
```

Comment:
`EVDEV_ABS_00=::42` is the syntax for "leave x/y as-is, just set resolution". It's preferable since it signals that the other bits are correct. Same thing for the other three please, thanks.

Reply:
fixed https://github.com/systemd/systemd/pull/13472/commits/ab6eaee0b0f9320db57e361e914168696b2db978

Reply:
sorry, I wasn't clear enough - you'll still need a line per axis (01, 35 and 35) but you can skip the first two values in each so they all have a value of `::$resolution`, with the resolution being the one for that axis like you had in the original commit.

Reply:
Changes added